src/axom/sidre/core/MFEMSidreDataCollection.cpp

```diff
@@ -242,6 +242,7 @@ void MFEMSidreDataCollection::createMeshBlueprintStubs(bool hasBP)
     m_bp_grp->createGroup("coordsets");
     m_bp_grp->createGroup("topologies");
     m_bp_grp->createGroup("fields");
+    m_bp_grp->createGroup("qfields");
   }
```

Comment:
`qfields` is not a part of the blueprint, so this part of the tree won't be recognized elsewhere. 

(It won't cause any blueprint verify errors, but it won't be checked for, or consumed)

Can we have the quadrature funcs be a field with a specific basis type?

Reply:
or otherwise an extension to a field?

Reply:
So, quadrature functions I, personally, would consider separate from a field variable / basis function. Since, they exist directly at the quadrature points and don't depend on a basis type or field/nodes. For example, if you look at how `mfem` creates them they only require a mesh, the integration order, and vector dimension of the quadrature function in order to be defined. Based on this, do you have an idea for what might be the best to handle such a thing within blueprint?

Reply:
In blueprint speak, a `field` is something that loosely encapsulates values  associated with a mesh topology. That still seems appropriate for a quadrature function, or that a quadrature function could be a new variant of a blueprint field.

One way to think about this, is how would delineate between this new construct in the overall structure: 

https://llnl-conduit.readthedocs.io/en/latest/blueprint_mesh.html 

Many of the same relationships that represent a field would have to be duplicated. Things like index generation would need to be duplicated, vs reused automatically.

Reply:
Okay, I think I'm starting to get what you're talking about. 

Then as far as the blueprint is concern, it might be possible to have it think of a `QuadratureFunction` as an L2 `GridFunction` with a `Gauss-Legendre` basis type, mfem::Ordering::Type::ByVDIM, and of sufficiently high enough order so its nodes correspond to the quadrature point locations of the `QuadratureFunction`. As long as internally, it doesn't try to project those points to a mesh's nodal locations, since you could have variables in a `QuadratureFunction` that need to remain non-negative. I guess even then you might want to have a flag associated with that field to denote it as a quadrature function.

Reply:
Yes -- if we can encode it in some how via a `basis` string name, thats great -- if not we can always add another key/value to specifically identify the QuadratureFunction case.

Reply:
@cyrush so I brought this up with the MFEM team just to see if they had any thoughts on the topic.

@v-dobrev had this to say:

> In terms of blueprint, a `QuadratureFunction` is another way of defining a field which is different from the way `GridFunctions` are defined. The information to store for a `QuadratureFunction` should be essentially what we save in `QuadratureFunction::Save` — `QuadratureSpace::Save` + `VDim` + data. Note that in the future, `QuadratureSpace::Save` will probably support types other than `Type: default_quadrature`.

@joshessman-llnl then suggested just calling the basis: `QF_Default_[ORDER]` which I like due to it being simple.

Based on their response, let me know what you think of the proposed `basis` string name.

Reply:
that sounds great to me, for folks looking to a reconstruct  QuadFunc, QuadSpac  - if we can have a way to pass that string like we do to mfem for a FiniteElementCollection, that would complete the circuit.

Reply:
@cyrush I'll work on the necessary PR for that within MFEM and link the PR here. Also, I think if we wanted to properly encode everything within the basis string like its done in the `FiniteElementCollection` the basis string here should be `QF_Default_[ORDER]_[VDIM]`. So, the `QuadratureSpace` would be able to take that name in and just make use of the `QF_Default_[ORDER]` part and supplied mesh to reconstruct itself. The `QuadratureFunction` could take the name in and just use the `[VDIM]` part as long as it's supplied an appropriate `QuadratureSpace`.

Nonetheless, I'll work on updating everything for this PR over from being `qfields` to `fields` for the underlying `blueprint`.

Reply:
Sounds good!